src/index.ts

```diff
@@ -343,6 +349,19 @@ export class ProjectConfig {
         this.builds = this.builds.map(applyBuildPreset);
       }
     }
+
+    /**
+     * autoBasePath
+     */
+    if (options.autoBasePath) {
+      this.autoBasePath = options.autoBasePath;
+      if (Array.isArray(this.builds)) {
```

Comment:
I think something like this should be fine:

```ts
for (const build of this.builds || []) {
  build.basePath = true;
}
```

The methods that read `polymer.json` run the JSON schema validator before calling this constructor, so I think we can assume types are safe.

Reply:
Ok! I had been inspired by the builds section. Updating...